maint.mk

```diff
@@ -1634,7 +1634,7 @@ _gl_TS_unmarked_extern_vars ?=
 _gl_TS_var_match ?= /^(?:$(_gl_TS_extern)) .*?(\w+)(\[.*?\])?;/
 
 # The names of object files in (or relative to) $(_gl_TS_dir).
-_gl_TS_obj_files ?= *.$(OBJEXT)
```

Comment:
this seems as a false positive.  `$(OBJEXT)` (object + extension) is defined by automake